auth/src/main/AndroidManifest.xml

```diff
@@ -16,16 +16,16 @@
             android:value="@string/twitter_consumer_secret"/>
 
         <activity
-            android:name="com.firebase.ui.auth.ui.email.ConfirmRecoverPasswordActivity"
-            android:label="@string/title_confirm_recover_password_activity"
+            android:name=".KickoffActivity"
+            android:label="@string/default_toolbar_title"
             android:exported="false"
-            android:theme="@style/FirebaseUI.Dialog"/>
+            android:theme="@style/Theme.AppCompat.NoActionBar"/>
```

Comment:
Neither your solution nor mine works. Everything I tried had no effect; even the actionbar is still there. I think something is going on with the style not being applied properly.